modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/Bean/AppCreateRequest.java

```diff
@@ -20,6 +21,13 @@
     private String overview_trackingCode = "AM_-1305386558";
     private String roles = "";
     private String tags = "";
+    private String uritemplate_policyGroupIds;
+    private String uritemplate_javaPolicyIds;
+    private String uritemplate_policygroupid0="1";
+    private String uritemplate_policygroupid1="1";
+    private String uritemplate_policygroupid2="1";
+    private String uritemplate_policygroupid3="1";
+    private String uritemplate_policygroupid4="1";
```

Comment:
This code segment has been refactored and this code line is not in here any more.